desktop/src/args.rs

```diff
@@ -25,6 +28,23 @@ pub(crate) struct Args {
         help = "Path to the ROM that should be loaded into the emulator.",
     )]
     pub(crate) path_to_rom: PathBuf,
+
+    #[structopt(
+        long = "--breakpoints",
+        parse(try_from_str = "parse_breakpoint"),
+        requires = "debug",
+        help = "Breakpoint that is added to the debugger at the very beginning. Breakpoints are \
+            specified in hexadecimal.",
```

Comment:
> hexadecimal

An example for the exact format would be great! (e.g. `0x42` or `0x0042` or `42h` or `$42`, etc.)